src/wp-includes/class-wp-user.php

```diff
@@ -36,6 +36,9 @@
  * @property string $rich_editing
  * @property string $syntax_highlighting
  * @property string $use_ssl
+ * @property array $caps
+ * @property array $roles
+ * @property array $allcaps
```

Comment:
Let's go with the more specific type notation, as it's already used in other places in this class. Also noting that the `bool[]` annotation below is actually incorrect, so we should fix that.
```suggestion
 * @property array<string, bool> $caps
 * @property string[] $roles
 * @property array<string, bool> $allcaps
```